src/abstract_datasets.jl

```diff
@@ -19,9 +19,9 @@ end
 
 function Base.show(io::IO, ::MIME"text/plain", d::D) where D <: AbstractDataset
     recur_io = IOContext(io, :compact => false)
-    
```

Comment:
it would be good to avoid polluting PRs with this formatting changes, there should be some option in your editor to avoid doing this